sdcm/cluster_k8s/gke.py

```diff
@@ -478,10 +479,3 @@ def sort_by_index(node):
 
         instances = sorted(instances, key=sort_by_index)
         return instances
-
-    # pylint: disable=protected-access
-    def _create_node(self, instance, node_index, dc_idx):
-        node = super()._create_node(instance=instance, node_index=node_index, dc_idx=dc_idx)
-        node._instance_wait_safe(
```

Comment:
So this code path we added as part of the multi tenant support, and didn't tested it ?

Reply:
It worked at that time. Failures labels setting started appearing last Friday and all the time after it. So, I guess something has changed on the GKE server side, update or something else.
So, I just started using another solution that works "now".

Reply:
o.k.